play-json/shared/src/test/scala/ReadsSharedSpec.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2009-2016 Lightbend Inc. <https://www.lightbend.com>
+ */
+package play.api.libs.json
+
+import org.scalatest._
+
+class ReadsSharedSpec extends WordSpec with MustMatchers {
```

Comment:
The `shared` part of the original `ReadsSpecs` (without the tests about the JVM specific code)